index.html

```diff
@@ -267,7 +267,7 @@
 <li>If the <var>timeout</var> property is present in <var>options</var> and has a positive value:
 <ol type='a'>
   <li>Wait for <var>timeout</var> milliseconds.</li>
```

Comment:
Hmm, should there be a hook in here somewhere to cancel the timer steps if and when the callback is executed before these timers fire?